app/views/lessons/_lesson.html.erb

```diff
@@ -0,0 +1,204 @@
+<div class='lesson-wrapper'>
+
+  <div>
+    <h1><%= @lesson.title %></h1>
+  </div>
+
+  <% if @lesson.example_image.exists? %>
+  <div class="featured-artist">
+    <div class="row">
+      <div class="example col-md-12 vcenter" id="example">
+        <div class="student-image">
+          <%= image_tag @lesson.example_image.url(:medium) %>
+        </div>
+        <div class="superhero-quote" id='synopsis'>
+          <%= @lesson.synopsis %>
+        </div>
+      </div>
+    </div>
+  </div>
+  <% end %>
+
+  <blockquote class="backdrop toc lessons-toc">
+    <p class="toc-title">Lesson Map</p>
+  </blockquote>
+
+  <hr />
+  <div id="overview">
+    <h1 class="overview-head heading anchor">Overview</h1>
+    <h2 id='objective' class="overview-heading anchor">objective</h2>
+    <p><%= @lesson.objective %></p>
+    <h2 id='setup' class="overview-heading anchor">Setup</h2>
+    <p><%= @lesson.setting_up %></p>
+    <h2 id='materials' class="overview-heading anchor">Materials</h2>
+    <p><%= @lesson.materials %></p>
+    <h2 id='photocopies' class="overview-heading anchor">Photocopies</h2>
+    <p>
+      <% if @lesson.first_photocopy.present? %>
+        <p><%= @lesson.first_photocopy_header %></p>
+        <p class="pdf-indent"><%= link_to @lesson.first_photocopy_file_name, @lesson.first_photocopy.url %></p>
+      <% else %>
+        <%= "N/A" %>
+      <% end %>
+      <% if @lesson.second_photocopy.present? %>
+      <p><%= @lesson.second_photocopy_header %></p>
```

Comment:
@jasonoverby So once we get a better attachment system, instead of having to check `present?` on each image, we can just loop through all the items in the collection.